spec/subnets-lifecycle.md

```diff
@@ -0,0 +1,68 @@
+# Subnet Lifecycle
+A subnet is a new subsystem that a user can spawn from a parent subnet in a permissionless and on-demand way, depending on scalability requirements. Subnets have separate consensus algorithms and cryptoeconomic rules from their parent subnet. Subnets are firewalled from the parent network.
+
+Subnets begin with a chosen "rootnet". In case of L2 subnets, “rootnets” refer to a layer 1 blockchain, such as Filecoin or Ethereum. Child subnets are spawned from the rootnet and the rootnet becomes the parent subnet.
+
+Each subnet can have any number of child subnets, while each child subnet only has one parent subnet. Subnets can scale infinitely, to layer 2 and beyond. A single hierarchy tree begins at the chosen rootnet.
+
+Subnets within a single hierarchy tree have native communication protocols and are able to transfer assets and state without a custom bridge.
```

Comment:
> have native communication protocols

Between which parties? I guess it's between all the subnets (parents, children, etc), I guess makes sense to point out what exactly these native communication protocols unite.

Reply:
My understanding is the communication does not really have to be specified in the spec, up to the implementation.